packages/kit/src/views/Setting/pages/Tab/DevSettingsSection/DeviceToken/index.ts

```
@@ -0,0 +1 @@
+export * from './DeviceToken';
```

ESLint

[error] Insert `⏎`

(prettier/prettier)